app/models/manageiq/providers/redhat/infra_manager/vm_import.rb

```diff
@@ -116,4 +158,22 @@ def select_host(vm)
   def password_auth(provider)
     provider.authentication_userid_passwords.first
   end
+
+  def get_first_vm_network_id(conn, dc_id)
+    conn.system_service.data_centers_service.data_center_service(dc_id)
+        .networks_service.list.select { |net| net.usages.include? 'vm' }.first.id
+  end
+
+  def get_vnic_profile_id_by_network_id(conn, network_id)
+    conn.system_service.networks_service.network_service(network_id).vnic_profiles_service.list.first.id
```

Comment:
Like in the previous comment, I'd suggest to sort the profiles by name, to make the result predictable.